packages/content-handler/src/websites/github-handler.ts

```diff
@@ -35,7 +35,7 @@ export class GitHubHandler extends ContentHandler {
     if (twitterTitle && twitterTitleContent) {
       twitterTitle.setAttribute(
         'content',
-        twitterTitleContent.replace(/GitHub - .*\//, '')
+        twitterTitleContent.replace(/GitHub - (.*?)\//, '')
```

Comment:
Nice. Could there be a slash in the `owner` name?

Reply:
eek, good catch. I did just look it up and they can not `All characters must be either a hyphen ( - ) or alphanumeric.
`

Reply:
Thanks!